model.py

```diff
@@ -11818,10 +11857,11 @@ def restrict_to_ready_deliverable_works(
                 or_(LicensePool.licenses_owned > 0, LicensePool.open_access)
         )
 
-        # Only find books in an appropriate collection.
-        query = query.filter(
-            LicensePool.collection_id.in_(collection_ids)
-        )
+        if collection_ids is not None:
```

Comment:
This seems like a significant and unrelated change if there's actually a case when `collection_ids` is None.